src/disco/components/Addon/index.js

```diff
@@ -61,6 +62,7 @@ export class AddonBase extends React.Component {
     iconUrl: PropTypes.string,
     install: PropTypes.func.isRequired,
     installTheme: PropTypes.func.isRequired,
+    isAddonEnabled: PropTypes.bool,
```

Comment:
that's a function, no? maybe the name of the function is not good enough as it implies a boolean.

Reply:
hmm..good point..